launch/all.launch

```diff
@@ -10,6 +10,7 @@
     <arg name="gps_file" default="" />
     <arg name="goal_file" default="" />
     <arg name="wind_file" default="" />
+    <arg name="num_ais_ships" default="5" />
```

Comment:
This is good, but you need to add something like
```
      <arg name="num_ais_ships" value="$(arg num_ais_ships)"/>
```
in the launch_all_mocks.launch tag